docs/io/fits/index.rst

```diff
@@ -724,6 +724,19 @@ the 3rd argument is not a header, it (and other positional arguments) are
 assumed to be the extension specification(s). Header and extension specs can
 also be keyword arguments.
 
+The :func:`printdiff` function will print a difference report of two fits files,
+including headers and data. The first two arguments must be two fits
+filenames or fits file objects with matching data types (i.e., if using strings
+to specify filenames, both input1 and input2 must be strings).  The third
+argument is an optional extension specification, with the same call format
+of :func:`getheader` and :func:`getdata`.  In addition you can add any keywords
+accepted by the :class:`FITSDiff` class::
+
+  >>> printdiff('inA.fits', 'inB.fits', ext=2)  # get a difference report of extension
+                                           # 2 of inA and inB
```

Comment:
Instead of breaking up comments, it is cleaner to have an one-line comment before the code.